### PR TITLE
support ttc

### PR DESCRIPTION
**Pull request by adipose**

### CODE REVIEW

File: src/mpc-hc/MainFrm.cpp [hunk 1]
```diff
@@ -16472,19 +16472,13 @@ bool CMainFrame::LoadSubtitle(CString fn, SubtitleInput* pSubInput /*= nullptr*/
             WIN32_FIND_DATA fd = {0};
             HANDLE hFind;
             
-            hFind = FindFirstFile(path + L"*.ttf", &fd);
+            hFind = FindFirstFile(path + L"*.*", &fd);
```
Comment (clsid2) [inline src/mpc-hc/MainFrm.cpp]: `*.?t?`

Comment (adipose) [reply src/mpc-hc/MainFrm.cpp]: Also .otf

Comment (adipose) [reply src/mpc-hc/MainFrm.cpp]: > `*.?t?`

Yeah that should work. There may be other font types to support later.

Comment (adipose) [reply src/mpc-hc/MainFrm.cpp]: https://learn.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-addfontresourceexa

Other possible extensions.  I don't think we'd use any.

Comment (clsid2) [reply src/mpc-hc/MainFrm.cpp]: No. Current three should cover everything.

**Comment by adipose** [comment]
OK, that's done!

**Pull request merged**